### DISCUSSION TITLE
Feature matrix for releases?

### DISCUSSION THREAD

**Discussion by gdevenyi** [openzfs/zfs#13252] [Q&A] [answered]
Assuming all other bugfixes are fixed at the minor release level, is there a feature matrix comparing 0.8.x (yes, I know its not longer getting bugfixes..) 2.0.x and 2.1.x?

I have a "major upgrade" to prepare for since 0.8.x is abandoned and I'm wondering whether I should go to 2.0.x or 2.1.x

**Comment by behlendorf** [comment, marked as answer]
There is a [feature flag matrix](https://openzfs.github.io/openzfs-docs/Basic%20Concepts/Feature%20Flags.html) in the OpenZFS docs.  This covers most of the major features, but it does omit a few features which didn't require an on-disk format change.

I'd recommend using the 2.1.x releases.
